code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -158,8 +158,14 @@
 	name = "guise of Istasha"
 	desc = "An odd amulet formed out of multiple floating parts, strung togethere by forces from another world."
 	icon_state = "eye_medalion"
-	trait = TRAIT_DIGINVIS
 
+/obj/item/clothing/neck/eldritch_amulet/guise/equipped(mob/user, slot)
+	. = ..()
+	user.AddElement(/datum/element/digitalcamo)
+
+/obj/item/clothing/neck/eldritch_amulet/guise/dropped(mob/user)
+	. = ..()
+	user.RemoveElement(/datum/element/digitalcamo)
```

Comment:
I'm gonna be honest in that I don't know what this amulet is, but...

This will make it obvious to non-silicons when somebody's wearing it, stating that their `skin seems to be shifting and morphing like is moving around below it.`. Is this the case currently? If not, maybe the element should be made `BESPOKE`, with an argument that decides if the examine message should be shown to non-silicons?

Also, missing empty line after the proc.

Reply:
Its some random item a bee contributor added. 

I'm also gonna be honest, I have no idea what the hell `BESPOKE` means

Reply:
Bespoke elements can have multiple instances based on the initialization arguments. You'd add the `ELEMENT_BESPOKE` flag to the element, say that it should start checking from the second argument, and make the second argument be a boolean for if there should be a message. If you do that, there will be separate instances of the element for cases where it does say the message, and for where it doesn't.

Reply:
Heretic amulet used for thermal vision